mos_tests/neutron/python_tests/test_dvr.py

```diff
@@ -17,6 +17,7 @@
 import pytest
 
 from mos_tests.neutron.python_tests import base
+from neutronclient.common.exceptions import NeutronClientException
```

Comment:
should be:
 import pytest

 from neutronclient.common.exceptions import NeutronClientException

 from mos_tests.neutron.python_tests import base